polyfills/Object/values/config.json

```diff
@@ -0,0 +1,27 @@
+{
+	"aliases": [
+		"es2017"
```

Comment:
Is ES2017 technically ES8?  I'm losing track.  We currently have a lot of polyfills aliased to `es6`, but not many attached to the year-based names and none attached to `es7` or `es8`.  We might need some policy here, @JakeChampion - thoughts?